src/lumigo_tracer/parsers/event_parser.py

```diff
@@ -82,10 +94,52 @@ def parse(event) -> Dict:
         return new_event
 
 
+class SNSHandler(EventParseHandler):
+    @staticmethod
+    def is_supported(event) -> bool:
+        if safe_get(event, ["Records", 0, "EventSource"]) == "aws:sns":
```

Comment:
```suggestion
        return safe_get(event, ["Records", 0, "EventSource"]) == "aws:sns"
```
Isn't pycharm suggested that to you?